sagemaker-python-sdk/mxnet_gluon_mnist/mxnet_mnist_with_gluon.ipynb

```diff
@@ -6,8 +6,19 @@
    "source": [
```

Comment:
<p>Link is not properly rendering? I don't know if it is an issue though, ReviewNB renders weird sometimes</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3319/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>